src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -73,6 +73,11 @@ interface Props {
   isDrawerOpen?: boolean
   /** Reflects the area not covered by navigation bars, tab bars, toolbars, and other ancestors  */
   safeAreaInsets: SafeAreaInsets
+  /** Error from Relay (MapRenderer.tsx). Needed here to send over the EventEmitter. */
+  relayError?: {
+    error: Error
+    retry: () => void
+  }
```

Comment:
I'm sure you were already planning to do so, but just in case; create a named interface for this and use that in all places you are typing this out.

Reply:
Yeah definitely – was going to put it in `types.ts`.